jobs/idp/spec

```diff
@@ -259,3 +260,9 @@ properties:
   idp.jaas.database.useBcrypt:
     description: "Does the password column use BCrypt for password storage"
     default: "true"
+  idp.totp.database.userSeedTable:
+    description: "Table used to store users for authenication"
+    default: "seed_table"
+  idp.totp.database.seedColumn:
```

Comment:
Seem to be missing `idp.totp.database.userColumn` from the `totp-authn-config.xml.erb`